targets/TARGET_RDA/TARGET_UNO_91H/device/TOOLCHAIN_GCC_ARM/rda_retarget.c

```diff
@@ -0,0 +1,27 @@
+/* mbed Microcontroller Library - sbrkwrap
+ * Copyright (C) 2009-2018 ARM Limited. All rights reserved.
```

Comment:
Can you edit this license to be as in other files

Reply:
done.